docs/dev/isolation.md

```diff
@@ -45,3 +45,7 @@ TODO
 
 ## Multi-tenancy
 We do not yet support multi-tenancy, in the sense that different tenants of the same server get isolated performance guarantees. When we do support this, it will need to be documented here.
+
+## Preserving isolation across RPC calls
+ScyllaDB uses RPC verbs heavily to communicate with other nodes. RPC is based on "verbs". Each RPC call will invoke a verb, which is associated with a C++ function. Most verbs have a fixed scheduling groups associated with them. The remote node, when receiving that RPC verb, will execute it with the scheduling group associated with said verb.
```

Comment:
Maybe you can refer to a file or function or something which has that association in case somebody wants to find the details?
Is this the commit 8c993e0728508728c2d5cb0c4323728831d84890 I referred in the TODO?

Reply:
Added reference to `messaging_service.cc`.